src/sql/workbench/services/connection/browser/cmsConnectionWidget.ts

```diff
@@ -97,11 +97,8 @@ export class CmsConnectionWidget extends ConnectionWidget {
 		let newAuthTypes = authTypeOption.categoryValues;
 		if (authTypeChanged) {
 			newAuthTypes = authTypeOption.categoryValues.filter((option) => option.name !== AuthenticationType.SqlLogin);
-		}
-		if (this._authTypeSelectBox) {
+			authTypeOption.defaultValue = AuthenticationType.Integrated;
```

Comment:
You've verified that changing this value doesn't affect the normal connection dialog?

Reply:
Normal connection dialog uses connectionWidget class instead.

Reply:
I'm just saying verify that the options map isn't shared across the widgets or something weird like that.